scripts/update-content-blocker.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const fs = require('fs');
+const fetch = require("node-fetch");
+
+(async function () {
+  const adList = await (await fetch("https://cdn.cliqz.com/adblocker/configs/safari-ads/allowed-lists.json")).json();
+
+  // Ad Network rules
+  const adNetworkRulesUrl = adList.safari.network;
+  const adNetworkRules = await (await fetch(adNetworkRulesUrl)).text();
+  fs.writeFileSync('content-blocker-lib-ios/Lists/safari-ads-network.json', adNetworkRules);
+
+  // Ads Cosmetic rules
+  const adCosmeticRulesUrl = adList.safari.cosmetic;
+  const adCosmeticRules = await (await fetch(adCosmeticRulesUrl)).text();
+  fs.writeFileSync('content-blocker-lib-ios/Lists/safari-ads-cosmetic.json', adCosmeticRules);
+
+  const trackingList = await (await fetch("https://cdn.cliqz.com/adblocker/configs/safari-tracking/allowed-lists.json")).json();
+
+  // Tracking Network rules
+  const trackingNetworkRulesUrl = trackingList.safari.network;
+  const trackingNetworkRules = await (await fetch(trackingNetworkRulesUrl)).text();
+  fs.writeFileSync('content-blocker-lib-ios/Lists/safari-tracking-network.json', trackingNetworkRules);
```

Comment:
Maybe it's overkill but you might be able to abstract this a bit?

```js
async function getRules(url, path) {
  const trackingNetworkRules = await (await fetch(trackingNetworkRulesUrl)).text();
  if (trackingNetworkRules !== '[]') {
    fs.promises.writeFile(path, trackingNetworkRules);
  }
}

async function getLists(config) {
  const trackingList = await (await fetch(`https://cdn.cliqz.com/adblocker/configs/${config}/allowed-lists.json"`)).json();
  await Promise.all([
    getRules(trackingList.safari.network, `content-blocker-lib-ios/Lists/${config}-network.json`),
    getRules(trackingList.safari.cosmetic, `content-blocker-lib-ios/Lists/${config}-cosmetic.json`),
  ]);
}

(async () => {
  await Promise.all([
    getLists('safari-ads'),
    getLists('safari-tracking'),
  ]);
})();
```

Reply:
Lets keep is simple for now :-)

Reply:
@chrmod I am happy to let you pick the definition of simplicity which suits you best!